content/doc/book/installing/index.adoc

```diff
@@ -238,6 +238,45 @@ If your `/etc/init.d/jenkins` file fails to start Jenkins, edit the `/etc/defaul
 Here, "8081" was chosen but you can put another port available.
 ====
 
+===== Installation of Java
+
+Jenkins requires Java in order to run, yet certain distros don't include this by default and  link:https://www.jenkins.io/doc/administration/requirements/java/[some Java version are incompatible] with jenkins.
+
+To install the Open Java Development Kit (OpenJDK) run the following:
+
+* Update the repositories 
+[source,bash]
+----
+sudo apt update
```

Comment:
small nit here. It may be helpful to explain the difference between `apt` and other commands so the user who may not be familiar with the difference actually knows. 
It may seem over the top for this type of thing but we want to help guide as much as possible the new user.
Thank you :-)

Reply:
Did it! Please do another review.